website/docs/quickstart-guide/docker.md

```diff
@@ -85,6 +85,51 @@ Find out more about:
 
 - [getting logs](../configuration/logging.md#docker-logs).
 
+### SQLite Setup with Docker Compose
+
+The following steps describe a production setup for SQLite:
```

Comment:
I don't think we should describe it as production. For example, that would be a production without backups